MonoGame.Framework.Content.Pipeline/Graphics/GraphicsUtil.cs

```diff
@@ -159,7 +167,9 @@ public static void CompressDxt(GraphicsProfile profile, TextureContent content,
 
             if (alphaRange == AlphaRange.Opaque)
                 Compress(typeof(Dxt1BitmapContent), content, generateMipMaps);
-            else if (alphaRange == AlphaRange.Cutout || sharpAlpha)
+            else if (sharpAlpha)
+                CompressFontDXT3(content, generateMipMaps);
```

Comment:
I think `sharpAlpha` is a bad way to trigger this special DXT3 compressor.  Technically it has nothing to do with the sharpness of alpha.  It is simply an optimization for perfectly monochrome (I guess actually white) textures that only vary in the alpha channel.

So I see three different approaches here:

 1. We add another `bool` to the `TextureProfile.PlatformCompressTexture`.
 2. We add a new `TextureProfile.PlatformCompressSpriteFontTexture()` and directly use it from `SpriteFont`.
 3. We internally can scan the texture data and automatically detect that this special DXT3 format is best.

I'm leaning towards 2.

But if we could do 3 right... it *might* be more useful...  or maybe it causes trouble when you don't expect it.